modules/datastore/tests/src/Unit/Storage/DatabaseTableTest.php

```diff
@@ -17,12 +17,105 @@
 use Psr\Log\LoggerInterface;
 
 /**
+ * @covers \Drupal\datastore\Storage\DatabaseTable
  * @group dkan
```

Comment:
Also needs `@coversDefaultClass \Drupal\datastore\Storage\DatabaseTable` in order to get the`@covers ::method` in the test methods.